packages/react-components/src/molecules/__tests__/UsersList.test.tsx

```diff
@@ -42,3 +42,35 @@ it('does not link external users', () => {
   );
   expect(getByText(/John Doe/).closest('a')).toBeNull();
 });
+
+describe('maximum users', () => {
+  it('truncates user list when maximum exceeded and shows additional count', () => {
+    const { getAllByRole, getByText } = render(
+      <UsersList
+        max={3}
+        users={Array.from({ length: 4 }).map(
+          () =>
+            ({
+              displayName: 'John Doe',
+            } as ExternalAuthor),
+        )}
+      />,
+    );
+    expect(getAllByRole('listitem').length).toEqual(4);
+    expect(getByText('+1')).toBeVisible();
```

Comment:
Should we first getByRole the 'img', and then check the text? I don't see it breaking that easily, but would be better.